modules/manage/pages/manage-buckets/edit-bucket.adoc

```diff
@@ -44,7 +44,25 @@ To display the user-interface for editing, left-click on the *Edit* button:
 image::manage-buckets/editBucketButton.png[,260,align=left]
 
 This displays the [.ui]*Edit Bucket Settings* dialog, which permits changes to be made to a subset of existing settings.
-All the settings contained here are described in detail for the [.ui]*Add Data Bucket* dialog, on the page xref:manage-buckets/create-bucket.adoc[Create a Bucket]
+All the settings contained here are described in detail for the [.ui]*Add Data Bucket* dialog, on the page xref:manage-buckets/create-bucket.adoc[Create a Bucket].
+
+[#edit-bucket-for-eccv]
+=== Edit a Bucket to Enable Cross Cluster Versioning 
+
+[.ui]*Edit Bucket Settings* dialog displays the additional setting, *Enable Cross Cluster Versioning*. Use this setting to enable the xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning] bucket property.
+
+image::manage-buckets/edit-bucket-with-eccv.png[,399,align=center]
+
+When you enable the bucket setting [.ui]*Enable Cross Cluster Versioning*, for each document processed by XDCR, XDCR stores additional metadata for the document in the extended attributes.
+This metadata is also called xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#hlv-data-maintained-in-xattr[Hybrid Logical Vector (HLV)].
```

Comment:
This metadata is called xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#hlv-data-maintained-in-xattr[Hybrid Logical Vector (HLV)], or version vectors.